### ISSUE TITLE
Add FASTQ inputs to the RNAseq pipeline

### ISSUE THREAD

**Issue by susannasiebert** [griffithlab/immuno-planning#1]

**Comment by gschang** [comment]
John has updated the rnaseq.CWL workflow to take FASTQ input files below. He and I are testing this update with HCC1395 data. Once done, I am going to further discuss with him, Susanna, and Jasreet. Next, we plan to make a PR to update the main branch.

# johnegarza/analysis-workflows
> https://github.com/johnegarza/analysis-workflows/tree/immuno_fastq_updates

**Comment by susannasiebert** [comment]
Do we need to also add these changes to https://github.com/genome/analysis-workflows/blob/master/definitions/pipelines/rnaseq_star_fusion.cwl since star fusion is what we will want to ultimately use?

**Comment by gschang** [comment]
This is a good point. We will add transcript fusion analysis to the next discussion agenda with John, Susanna, and Jasreet. Let's further discuss how to proceed for star-fusion option or any other plan as Malachi and Chris suggest.

As John suggested, I have looked into his updates for any other inputs. I think that it is fine to make a PR for this update (rnaseq FASTQ input support) at first, then next for star-fusion (at https://github.com/griffithlab/immuno-planning/issues/2 as Susanna created). I am now looking at new rnaseq results that John made for test, and I will update here again.

**Comment by gschang** [comment]
As discussing with John and Jasreet today, I have checked the results generated by the new rnaseq.CWL workflow that John updated for FASTQ input format support. In brief, the new rnaseq.CWL workflow works as intended and all the results are all right.

In detail,

(1) Two main changes

These changes that John made are what we need. As John already checked, I doublechecked that these new workflow updates are working as intended.

../subworkflows/bam_to_trimmed_fastq_and_hisat_alignments.cwl
../tools/sequence_to_fastq_rna.cwl            # replacing bam_to_fastq.cwl

(2) Results comparison

The new rnaseq.CWL results are almost identical to the previous rnaseq results, which John generated both. RNA metric was as expected and the transcript abundance results were identical.

As discussing with @John Garza and @jhundal, let's make a PR to update the master branch if you agree. As you see on-going discussion, the next discussion will be (1) implementation of transcript fusion analysis into rnaseq.CWL workflow and (2) immuno workflow test with a full dataset.
